public/content/lessons/2016/cross-products/index.mdx

```diff
@@ -1,10 +1,253 @@
+
+<Figure
+    image="./figures/compute-with-determinant/ComputeCrossProduct.svg" 
+/>
+
+This is because a matrix whose columns represent $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$ corresponds with a linear transformation that moves the basis vectors i hat and j hat to $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$. 
+
+<Figure
+    image="./figures/compute-with-determinant/ChangeOfArea.svg"
+    video="./figures/compute-with-determinant/ChangeOfArea.mp4"
+    show="video" 
+/>
+
+The determinant is all about measuring how areas change due to a transformation, and the prototypical area to look at is the unit square resting on i hat and j hat. After the transformation, that square turns into the parallelogram we care about. So the determinant, which generally measures the factor by which areas are changed, gives the area of this parallelogram, since it evolved from a square that started with area 1. 
+
+<Figure
+    image="figures/determinant-basis-vectors-transformed-orientation.png" 
+    caption="TODO: replace with image callback from chapter 5"
+/>
+
+What's more, if $\vec{\mathbf{v}}$ is to the left of $\vec{\mathbf{w}}$, it means orientation is flipped during the transformation, which is what it means for a determinant to be negative.
+
+<Figure
+    image="./figures/compute-with-determinant/DeterminantNegative.svg"
+    video="./figures/compute-with-determinant/DeterminantNegative.mp4"
+    caption="Since the orientation is flipped, the determinant is negative."
+/>
+
+<Figure
+    image="./figures/compute-with-determinant/DeterminantPositive.svg"
+    video="./figures/compute-with-determinant/DeterminantPositive.mp4"
+    caption="Since the orientation isn't flipped, the determinant is positive."
+/>
+
+### Determinant example
+
+<Figure
+    image="./figures/determinant-example/DeterminantExample.svg" 
+/>
+
+As an example, if $\vec{\mathbf{v}}$ has coordinates $\begin{bmatrix}-3\\1\end{bmatrix}$, and $\vec{\mathbf{w}}$ has coordinates  $\begin{bmatrix}2\\1\end{bmatrix}$, the determinant of the matrix with those coordinates has columns is $(-3)\cdot1 - 2\cdot1$, which is $-5$. So evidently the area of the parallelogram they define is $5$, and since $\vec{\mathbf{w}}$ is to the right of $\vec{\mathbf{v}}$ it should make sense that this value is negative.
+
+<Figure
+    image="./figures/determinant-example/DeterminantTransformation.svg" 
+    video="./figures/determinant-example/DeterminantTransformation.mp4"
+/>
+
+I'd recommend playing around with this notion a bit in your head to get an intuitive feel for the cross product.
+
+<Question
+  question="What is result of of $\vec{\mathbf{v}} \times \vec{\mathbf{w}}$ where $\vec{\mathbf{v}} = \begin{bmatrix}1\\-2\end{bmatrix}$ and $\vec{\mathbf{w}} = \begin{bmatrix}4\\3\end{bmatrix}$?"
+  choice1="5"
+  choice2="-11"
+  choice3="11"
+  choice4="10"
+  answer={3}
+>
+
+We can start by visualizing the two vectors in space:
```

Comment:
Although it's nice to have the visuals, I wouldn't want to imply to the reader that they need to visualize it. Maybe first just show the straightforward way to answer, maybe with some reminder of a way to mentally picture the determinant computation.

Then, since you already made all the imagery, you could add below something like "We can gut check our answer by drawing the diagram, and seeing if 11 feels like a reasonable area, and whether it makes sense that it's positive"

Reply:
I agree that changing it to a straightforward answer makes sense. I think I was trying to be mindful of the fact that in the "standard view" section we go away from computing the cross-product as the determinant.